app/src/Layout/Layout.css

```diff
@@ -23,11 +23,30 @@
   color: white;
   font-size: 2vw;
   font-weight: 400;
-  margin-right: 2%;
+  margin-right: 25px;
```

Comment:
Not gonna request changes over something with this little an effect, but why change this when the Logo on the left has a margin of 2%? I'm not very opposed to it being fixed, but shouldn't they at least be consistent, you know?